help.go

```diff
@@ -237,6 +237,11 @@ func DefaultCompleteWithFlags(ctx context.Context, cmd *Command) {
 		lastArg = args[argsLen-1]
 	}
 
+	if len(cmd.runArgs) > 1 && cmd.runArgs[len(cmd.runArgs)-1] == "-" {
```

Comment:
This looks like an equality check for `"-"` but then `lastArg` is later checked for a _prefix_ of `"-"`. Is that right?

Reply:
I dont think so. This is mainly for tab completion after user hits tab after entering a '-'. Let me double check. I might need to add more tests